<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>정밀검사</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>slow.png</normaloff>slow.png</iconset>
  </property>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_path2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>폴더선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>탐지 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_find">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>891</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>파일선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_pass">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_button">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);</string>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="file_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>891</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>정상 : </string>
   </property>
  </widget>
  <widget class="QProgressBar" name="loading">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_button2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
</string>
   </property>
   <property name="text">
    <string>경로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 77, 77);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>검사</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>path_find</zorder>
  <zorder>label_2</zorder>
  <zorder>path</zorder>
  <zorder>loading</zorder>
  <zorder>path_pass</zorder>
  <zorder>label_3</zorder>
  <zorder>find_path2</zorder>
  <zorder>file_path</zorder>
  <zorder>textBrowser_3</zorder>
  <zorder>check</zorder>
  <zorder>find_path</zorder>
  <zorder>finish_button</zorder>
  <zorder>finish_button2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
